examples/c/lsm.bpf.c

```diff
@@ -0,0 +1,19 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <errno.h>
```

Comment:
Spoke too soon, see https://github.com/libbpf/libbpf-bootstrap/actions/runs/7628266626/job/20828204493?pr=245. I think it's because of errno.h include. I think it's better to just `#define -EPERM` explicitly and avoid including user-space headers like errno.h

Reply:
@anakryiko Although it's slightly regrettable, I feel success is just around the corner. Adjustments have been made, and we can trigger the check again. Thank you.